src/wallet/rpcwallet.cpp

```diff
@@ -3323,10 +3323,16 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             "         \"CONSERVATIVE\""},
                         },
                         "options"},
+                    {"add_to_wallet", RPCArg::Type::BOOL, /* default */ "true", "When false, returns the fee-bumped\n"
+    "transaction in PSBT format instead of submitting to the wallet and mempool.\n"
+    "The wallet will try to sign as much of the PSBT as possible, but may returned something\n"
+    "unsigned if there are watch-only transactions in the wallet."
+                    },
```

Comment:
This seems like it shouldn't be a positional parameter...

Reply:
fwiw that point was discussed above

Reply:
I don't understand the point above. We call `RPCTypeCheckObj` with `fStrict`=`true`, so unrecognised keys will error...

Reply:
Yeah I missed that the strict option when I left my comment and then @ryanofsky corrected me.

Reply:
+1 for moving to options dictionary.

It could also default to true for watch-only wallets, though not for external signer wallets (#16546), so maybe keep it simple :-)